docs/src/apimanual.md

```diff
@@ -821,6 +821,31 @@ MOI.read_from_file(src_2, filename)
 ```
 Note how the compression format (GZip) is also automatically detected from the filename.
 
+In some cases `src` may contain constraints that are not supported by the file format (e.g.,
+binary variables in the CBF format). If so, you should copy `src` to a bridged model using
```

Comment:
I would add that CBF supports integer hence even if it doesn't support binary, it can be bridged. Here it seems adding bridges solves everything, e.g. It could even make LP support PSD